package.json

```
@@ -65,11 +67,13 @@
     "jest": "^28.1.1",
     "jest-environment-jsdom": "^28.1.1",
     "lint-staged": "^13.0.2",
+    "nodemon": "^3.0.1",
+    "npm-run-all": "^4.1.5",
     "prettier": "^2.7.1",
     "sort-package-json": "^1.57.0",
     "start-server-and-test": "^1.14.0",
     "ts-jest": "^28.0.5",
-    "typescript": "^4.7.4",
+    "typescript": "^4.9.5",
```

One problem here, it looks like our Vite template is using an older version of `typescript` than the original starter repo did, and npm isn't going to automatically go back to an old version since that `^x.xx.xx` means "any version equal to or higher than x version"

to fix this, we'll have to 

1. revert this line in `package.json` 
```suggestion
    "typescript": "^4.7.4",
```

2. run a command to remove your local files and regenerate them: 
`rm package-lock.json; rm -rf node_modules; npm install`

---

then hopefully when you commit and push, we'll only have the differences that you added on purpose with the new packages, without a mismatch in `typescript` version 🙌